src/molecules/navbar/Navbar.tsx

```diff
@@ -8,16 +8,30 @@ export interface NavbarProps {
   children?: ReactElement
 }
 
-export const Navbar = ({
-  children,
-  ...logoProps
-}: NavbarProps): JSX.Element => {
+export const Navbar = ({ companyName, logo }: NavbarProps): JSX.Element => {
+  const [windowSize, setWindowSize] = React.useState([0, 0])
+  const isMobileAndKeyboardOpen = () => {
+    const isKeyboardOpen = windowSize[1] < 500
```

Comment:
[Q] I don't understand how this function determines whether the keyboard is open or not. It only seems to evaluate the window height. If the function doesn't directly determine whether the keyboard is open, then I would give your variable a different name.

Reply:
I've refactored the logic so it's more consistent and documented why I took this approach